### PR TITLE
Wing Wing & FW Fixes

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates the Wing Wing airframe doc. Originally was going to delete, but the frame is still good.

In particular:
1. I've ordered this last, reflecting that it is an older frame and very old flight controller.
1. Airframe is no longer directly selectable in QGC. This updates the airframe you can select to generic flying wing
1. Because the setup is generic, this also shows how to set up the actuators. 
1. I also tidied up some of the config pages.

I'm going to merge, but have added some questions for @sfuhrer which might result in post process.

### CODE REVIEW

File: en/frames_plane/wing_wing_z84.md [hunk 1]
```diff
@@ -71,8 +71,22 @@ The images below give a rough idea about the assembly process, which is simple a
...
-Select the Z-84 in the flying wing section of the QGC airframe config: 
+Select **Flying Wing > Generic Flying Wing** in the QGroundControl [Airframe Configuration](../en/config/airframe.md): 
 
 ![QGC - select firmware for West Wing](../../assets/airframes/fw/wing_wing/qgc_firmware_flying_wing_west_wing.png) 
+
+### Actuator Mapping
+
+Set up the [Actuator Configuration](../en/config/actuators.md) to match the wiring for the ailerons and throttle as [indicated above](#wiring).
+
+![QGC - set the actuators](../../assets/airframes/fw/wing_wing/qgc_actuator_config.png)
```
Comment (hamishwillee) [inline en/frames_plane/wing_wing_z84.md:86]: @sfuhrer Can you sanity check this image. It should match the config up in the wiring section.

Comment (sfuhrer) [reply en/frames_plane/wing_wing_z84.md:86]: Yes looks correct!

**Comment by sfuhrer** [comment]
Thanks, looks correct and clean

**Comment by hamishwillee** [comment]
Thanks @sfuhrer - much appreciated!

**Pull request merged**